tests/__snapshots__/singleTag.test.js.snap

```diff
@@ -5,7 +5,9 @@ exports[`single tag 1`] = `
 function fun(param0) {}
 
 export const SubDomain = {
-  /** @returns {import(\\"axios\\").AxiosResponse<import(\\"../types\\").SubDomain>} */
+  /**
+   * @returns {import(\\"axios\\").AxiosResponse<import(\\"../types\\").SubDomain>}
```

Comment:
As I said in the description of this PR:

> A single-line JSDoc comment (e.g. `/** @type {number} */`) will now only be created if the comment length does not exceed `printWidth`.

Reply:
This should be a single line, we created a single line if there is a single line in comment that is not about printwidth, we don't strict about printwidth, that line could be more than printwidth, that is not matter.

Reply:
The user specifically set the print width, so we should honor that if possible IMO. Otherwise, why have a print width in the first place?

Reply:
Or is there a reason why we should ignore print width?

Reply:
I removed the change.